src/Panel/TimeLayerSliderPanel/TimeLayerSliderPanel.tsx

```diff
@@ -0,0 +1,496 @@
+      endDate
+    } = this.state;
+
+    const resetVisible = true;
+
+    const startDateString = startDate ? startDate.toISOString() : undefined;
+    const endDateString = endDate ? endDate.toISOString() : undefined;
+    const valueString = value ? value.toISOString() : undefined;
+    const mid = startDate!.clone().add(endDate!.diff(startDate) / 2);
+    const marks = {};
+    const futureClass = moment().isBefore(value) ? ' timeslider-in-future' : '';
+    const extraCls = className ? className : '';
+    const disabledCls = timeAwareLayers.length < 1 ? 'no-layers-available' : '';
+
+    marks[startDateString] = {
+      label: startDate!.format(dateFormat)
+    };
+    marks[endDateString] = {
+      label: endDate!.format(dateFormat),
+      style: {
+        left: 'unset',
+        right: 0,
+        transform: 'translate(50%)'
+      }
+    };
+    marks[mid.toISOString()] = {
+      label: mid.format(dateFormat)
+    };
+
+    return (
+      <div className={`time-layer-slider ${disabledCls}`.trim()}>
+
+        <Popover
+          placement="topRight"
+          title={tooltips.dataRange}
+          trigger="click"
+          content={
+            <RangePicker
+              showTime={{ format: 'HH:mm' }}
+              defaultValue={[startDate, endDate]}
+              onOk={this.updateDataRange}
+            />
+          }
+        >
+          <SimpleButton
+            className="change-datarange-button"
+            icon="calendar-o"
+          />
+        </Popover>
+        {
+          resetVisible ?
+            <SimpleButton
+              type="primary"
+              icon="refresh"
+              onClick={this.setSliderToNow}
+              tooltip={tooltips.setToNow}
+            /> : null
+        }
+        <this._TimeLayerAwareSlider
+          className={`${extraCls} timeslider ${futureClass}`.trim()}
+          formatString={dateFormat}
+          defaultValue={startDateString}
+          min={startDateString}
+          max={endDateString}
+          value={valueString}
+          marks={marks}
+          onChange={this.onTimeChanged}
+        />
+        <div className="time-value">
+          {value.format('DD.MM.YYYY HH:mm:ss')}
+        </div>
+        <ToggleButton
+          type="primary"
+          icon="play-circle-o"
+          className={extraCls + ' playback'}
+          pressed={autoPlayActive}
+          onToggle={this.autoPlay}
+          tooltip={autoPlayActive ? 'Pause' : 'Autoplay'}
+          pressedIcon="pause-circle-o"
+        />
+        <Select
+          defaultValue={'1'}
+          className={extraCls + ' speed-picker'}
+          onChange={this.onPlaybackSpeedChange}
+        >
+          <Option value="0.5">0.5x</Option>
+          <Option value="1">1x</Option>
+          <Option value="2">2x</Option>
+          <Option value="5">5x</Option>
+          <Option value="10">10x</Option>
+          <Option value="100">100x</Option>
+          <Option value="300">300x</Option>
```

Comment:
It would be nice to have this list configurable.

Reply:
`autoPlaySpeedOptions` can be passed to the panel now. If these are not specified, default values are used.